static/scss/header.scss

```diff
@@ -0,0 +1,61 @@
+// sass-lint:disable mixins-before-declarations
```

Comment:
This stylesheet should probably be called `home.scss`, not `header.scss`, since that could be confused with the nav bar